modules/nf-core/software/samtools/stats/main.nf

```diff
@@ -1,19 +1,21 @@
 // Import generic module functions
-include { saveFiles; getSoftwareName } from './functions'
+include { initOptions; saveFiles; getSoftwareName } from './functions'
 
 params.options = [:]
+options        = initOptions(params.options)
 
 process SAMTOOLS_STATS {
     tag "$meta.id"
+    label 'process_low'
     publishDir "${params.outdir}",
         mode: params.publish_dir_mode,
-        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), publish_id:meta.id) }
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:meta, publish_by_meta:['id']) }
 
-    conda (params.enable_conda ? "bioconda::samtools=1.12" : null)
+    conda (params.enable_conda ? "bioconda::samtools=1.10" : null)
```

Comment:
why did you downgrade samtools here?

Reply:
mistake, I'll fix that in the next PR.
Thanks for spotting that

Reply:
cf https://github.com/nf-core/sarek/pull/386